pkgs/applications/editors/emacs/elisp-packages/manual-packages/consult-gh/default.nix

```diff
@@ -1,46 +1,33 @@
-{ lib
-, melpaBuild
-, fetchFromGitHub
-, consult
-, embark
-, forge
-, gh
-, markdown-mode
-, writeText
-, unstableGitUpdater
+{
+  lib,
+  consult,
+  embark,
+  fetchFromGitHub,
+  forge,
+  markdown-mode,
+  melpaBuild,
+  unstableGitUpdater,
+  writeText,
 }:
 
-let
-  commit = "3a07139a1f7e38b959ce177a122c8f47c401d7fa";
-in
 melpaBuild {
   pname = "consult-gh";
   version = "0.12-unstable-2024-04-23";
 
-  inherit commit;
-
   src = fetchFromGitHub {
     owner = "armindarvish";
     repo = "consult-gh";
-    rev = commit;
+    rev = "3a07139a1f7e38b959ce177a122c8f47c401d7fa";
     hash = "sha256-BZloG5RuQzC2RwCfvqPPhGcbsCabQWBnRHdU62rwNdo=";
   };
 
   packageRequires = [
     consult
     embark
     forge
-    gh
```

Comment:
It seems `gh` is needed at run time.

You can either use `propagatedUserEnvPkgs = [ gh ];` or patch the [source][1] to ensure `gh` is available.

[1]: https://github.com/armindarvish/consult-gh/blob/3a07139a1f7e38b959ce177a122c8f47c401d7fa/consult-gh.el#L39

Reply:
Done